.travis.yml

```
@@ -1,10 +1,21 @@
 language: java
 
 stages:
+  - test
   - name: release
-    if: branch == master AND fork == false AND type == api
+    if: env(RELEASE) == true AND fork == false
```

Hmm, we could actually get away with this:
```suggestion
    if: env(RELEASE) == true
```
The permissions are inherently checked by Artifactory and GitHub, we don't actually need to filter out forks here.
OTOH it might be worth leaving it as is, just to show the intent of the workflow.